_posts/2021-07-07-JdlC-meetup-June.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+image: '/img/mick-haupt-Spjvmx7MJGs-unsplash.jpg'
+title: "What Open Science can learn from Free Software"
+excerpt_separator: <!--break-->
+date: 2021-07-07
+---
+Photo by <a href="https://unsplash.com/@rocinante_11?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Mick Haupt</a> on <a href="https://unsplash.com/s/photos/wall-garden?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+Authors:  
+[Lieke de Boer](https://twitter.com/liekedb) / Netherlands eScience Center  
+
+[Javier de la Cueva](https://twitter.com/jdelacueva) does not like slideshows – &quot;_it limits the ways we communicate, like squeezing thoughts into 140 characters_&quot;, but for the NL-RSE meetup, he made one anyway. De la Cueva is a lawyer, philosopher, and Open Science advocate, with expertise in free software and intellectual property. During the meetup on Friday, 11 June 2021, he gave a passionate talk about what the Open Science movement can learn from the Free Software movement, blending the topics of software development and science in a way not unfamiliar to those identifying as RSEs.
+
+If anyone&#39;s an expert on these topics, it&#39;s De la Cueva. He switched from Windows to Linux in 2000, after receiving a Blue Screen of Death on his Windows machine – &quot;_Why would you create a wall that doesn&#39;t allow me to understand something I paid for?_&quot;. In 2002, when Wikipedia was planning to commercialize, De la Cueva and others [forked the Spanish version of the page onto a server in the University of Seville, Spain](https://www.wired.co.uk/article/wikipedia-spanish-fork), proving to Wikipedia&#39;s owners that the entire project could be run on non-profit servers – a move some consider responsible for keeping Wikipedia free of commercial ads and paywalls to this day. Later, in 2006, he was, in his own words, &quot;_the first lawyer_ _to obtain a judicial ruling where the term _[_copyleft_](https://copyleft.org/)_ was included_&quot;. For the case in question, [Ladinamo](https://edri.org/our-work/edrigramnumber4-9spaincc/), De la Cueva defended the owners of a bar who were sued for failing to pay the compulsory licensing fee for playing music in their establishment. He won the case by successfully arguing that the music&#39;s Creative Commons licenses allowed for public performance, and that no additional fees had to be paid.
```

Comment:
Not sure how you wanted your __copyleft__ to render, but I feel there might be one too many _ before the quote.